wdio.conf.js

```diff
@@ -1,16 +1,14 @@
 const path = require('path');
-const defaultWdioConfig = require('terra-toolkit/config/wdio/wdio.conf');
-
-const wdioConfig = defaultWdioConfig.config;
+const { config } = require('@cerner/terra-functional-testing/lib/config/wdio.conf');
 
 if (process.env.npm_package_name !== 'terra-graphs') {
-  wdioConfig.specs = [path.join(__dirname, 'packages', process.env.npm_package_name, 'tests', 'wdio', '**', '*-spec.js')];
+  config.specs = [path.join(__dirname, 'packages', process.env.npm_package_name, 'tests', 'wdio', '**', '*-spec.js')];
 }
 
 const travis = process.env.TRAVIS;
 
 if (travis) {
-  wdioConfig.host = 'localhost';
+  config.host = 'localhost';
 }
```

Comment:
we can remove this and update the travis.yml to not spin up a docker container.
the travis.yml file should look more like this: https://github.com/cerner/terra-framework/blob/main/.travis.yml